docs/compilation/composing_functions_with_modules.md

```diff
@@ -260,15 +260,65 @@ In this case, the policy states that the first output of the `collatz` function
 You can use the `fhe.Wire` between any two functions. It is also possible to define wires  with `fhe.AllInputs` and `fhe.AllOutputs` ends. For instance, in the previous example:
 ```python
     composition = Wired(
-        [
+        {
             Wire(AllOutputs(collatz), AllInputs(collatz))
-        ]
+        }
     )
 ```
 
 This policy would be equivalent to using the `fhe.AllComposable` policy.
 
-## Current limitations
+## Automatic module tracing
```

Comment:
cc @yuxizama for the doc review :pray:

Reply:
Shall I wait for more updates or that's all?